### ISSUE TITLE
Repo manager success feedback

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#1200]
A message returning the result of a write operation while modifying the registry would be helpful.

`A new dataset with id "abx23" has been added".

**Comment by diekhans** [comment]
as long as there is a quiet option.

large command line automations like to only see error messages

**Comment by jeromekelleher** [comment]
Agreed on both counts. I think we should add this functionality using the logging framework, so that these messages are at the INFO level. We should have a -q --quiet option that turns off all messages, and a -v --verbose that turns up the verbosity level. That is, -vvv will turn on all DEBUG messages,  etc. 

I'm adding this to the logging milestone to keep these issues batched together.
